react/week5/githubuser-search/src/Components/User/index.js

```diff
@@ -0,0 +1,2 @@
+import User from './User'
+export default User
```

Comment:
A bit overkill in my opinion

Reply:
all the credit goes to Mads, he thought us very well.